frontend/workflows/experimentation/src/view-experiment-run.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
```

Comment:
would be better to have Derek review the UI parts

Reply:
Could you take a look too - you have more context as when it comes to how this view should/could work. I can ask @dschaller  for a separate review when it comes to the correctness of React etc. but would prefer this PR not to be blocked on this.